src/notebookConcatDocument.ts

```diff
@@ -1,343 +1,559 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+
```

Comment:
@jakebailey @heejaechang @bschnurr

This is the new 'concat' document. It should be reusable by pylance.